### DISCUSSION TITLE
[Feature] Branding/White Label

### DISCUSSION THREAD

**Discussion by samschultzponsys** [immich-app/immich#13639] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The ability to change logos, and titles would be really nice, would totally see that as an option for paid license holders. Thanks!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by samschultzponsys** [comment]
To be more clear, something that is more integrated and GUI based as I see others have found ways to mess with the file base but I am looking for something more simple, maybe not even mess with color scheme but focus on text titles, favicons and logos.

**Comment by bo0tzz** [comment]
#1634, as requested in the template please search for existing requests.
